lib/flipper/ui.rb

```diff
@@ -41,15 +41,15 @@ def self.root
 
     def self.app(flipper = nil, options = {})
       env_key = options.fetch(:env_key, 'flipper')
+      rack_protection_options = options.fetch(:rack_protection, { use: :authenticity_token })
```

Comment:
I noticed you are changing the default options here from nothing to only using authenticity token. Why is that?

Reply:
No, I kept it as it was - some of rack protection middlewares, are disabled by default, unless explicitly enabled (e.g. `authenticity_token`), and some are enable by default, unless skipped: https://github.com/sinatra/sinatra/blob/master/rack-protection/lib/rack/protection.rb#L23-L51

So:
```ruby
builder.use Rack::Protection, { use: :authenticity_token }
```

has the same effect as:
```ruby
builder.use Rack::Protection
builder.use Rack::Protection::AuthenticityToken
```